test/mx/test_transform_equals.py

```diff
@@ -0,0 +1,219 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+from typing import List
+
+import pytest
+import numpy as np
+
+from gluonts.core import serde
+from gluonts import time_feature, transform
+from gluonts.core.component import equals
+from gluonts.dataset.field_names import FieldName
+from gluonts.dataset.loader import Batch
+from gluonts.model.canonical._estimator import CanonicalRNNEstimator
+
+
+def test_map_transformation():
+
+    tran1 = transform.VstackFeatures(
+        output_field="dynamic_feat",
+        input_fields=["age", "time_feat"],
+        drop_inputs=True,
+    )
+
+    tran2 = serde.flat.clone(tran1)
```

Comment:
I think I would write this as

```py
    assert equals(t, serde.flat.clone(t)
    assert not equals(t, serde.flat.clone(t, {"drop_inputs": False})
```
Or maybe shorten it to:

```py
    assert equals(t, clone(t))
    assert not equals(t, clone(t, {"drop_inputs": False}))
```

Reply:
For object can be created in 1 line, will change it to.
```Python
    assert equals(t, clone(t))
    assert not equals(t, clone(t, {"drop_inputs": False}))
```
For the object can not be created in 1 line, will use `tran = clone(...)`.